functions.php

```diff
@@ -7,3 +7,24 @@
  * @package Twenty Twenty-Five
  * @since Twenty Twenty-Five 1.0
  */
+
+/**
+ * Enqueue editor-style.css in the editors.
+ */
+function twenty_twenty_five_editor_style() {
+	add_editor_style( get_parent_theme_file_uri( 'editor-style.css' ) );
```

Comment:
In the codex page for [add_editor_style()](https://developer.wordpress.org/reference/functions/add_editor_style/) I see the following:

> If a child theme is used, both the current child and parent theme directories are tested and both the files with the same relative path are linked with this single call if they are found.

It was my understanding that it should work like that. But after testing it out, I can only get it working with the tweak you added. Maybe there's something that changed and it's not documented? Or maybe just a bug.

The other thing I was thinking of, is if we want to move this one out to the assets folder to keep things more organized.